src/API.js

```diff
@@ -18,6 +18,7 @@ class API extends ApplicationAPI {
       Routers.AlertsRouter,
       Routers.AllStopsRouter,
       Routers.DelayRouter,
+      Routers.FeedInfoRouter,
```

Comment:
should probably name this GTFSFeedInfo - not clear which feed this is, because we also have a live-tracking feed!